server/src/models/Flashcard.ts

```diff
@@ -51,7 +57,8 @@ function validateFlashcardUpdate(flashcard: typeof Flashcard): Joi.ValidationRes
         prompt: Joi.string().min(1).max(4096),
         imageUrl: Joi.string(),
         answer: Joi.string().min(1).max(4096),
-        extraInfo: Joi.string().min(1).max(4096)
+        extraInfo: Joi.string().min(1).max(4096),
+        collectionID: Joi.string()
```

Comment:
```suggestion
        collectionId: Joi.string()
```

Szczerze mówiąc nadal do mnie nie przemawia pomysł dodawania collectionId do pojedynczej fiszki. Wydaje mi się, że usuwanie fiszek powinno się odbywać z podaniem od razu kolekcji w uri

`router.delete('/:collectionId/:flashcardId', async (req: Request, res: Response) => {'`

ale też ekspertem nie jestem

Reply:
to, że flashcard przechowuje collectionId przydaje się też przy autoryzacji (jak się sprawdza, czy jest dostęp do fiszki/kolekcji w zależności od tego, czy kolekcja prywatna/publiczna). 
plus też nigdzie nie mamy zagnieżdżonych ścieżek, więc to by wymagało większej rewolucji...

Reply:
@CaptainObjective możesz proszę rozstrzygnąć, czy idziemy modelem id kolekcji w fiszce, czy nie? 
Bo jeśli tak, to bym poprosiła o approve i mogę wtedy wrzucić autoryzację (na razie mam dla fiszki) 
A jeśli nie (jak proponuje Gosia) i wtedy wprowadzę stosowne zmiany. 
Dzięki!

Reply:
Zgadzam się że route w stylu `router.delete('collection/:collectionId/flashcard/:flashcardId', async (req: Request, res: Response) => {` byłby bardziej w duchu REST my jednak to API robimy na nasze potrzeby więc nie musimy się aż tak trzymać konwencji. Co do posiadania odniesienia do kolekcji na pojedyńczej flashcardzie to pewnie da się to zrobić bez tego ale tak myślę czy w przyszłości nie chcielibyśmy sobie np. dzielić fiszek pomiędzy kolekcjami. Może być przecież taka sytuacja że to sama fiszka ma sens więcej niż jednej kategorii albo user który chce zrobić kolekcje fiszek z wykorzystaniem już istniejących.

Reply:
Ok. W takim razie proponuję jedynie zmianę literki z 'ID' na 'Id'